spec.html

```diff
@@ -46262,6 +46262,20 @@ <h1>get Promise [ @@species ]</h1>
           <p>Promise prototype methods normally use their *this* value's constructor to create a derived object. However, a subclass constructor may over-ride that default behaviour by redefining its @@species property.</p>
         </emu-note>
       </emu-clause>
+
+      <emu-clause id="sec-promise.withResolvers">
+        <h1>Promise.withResolvers ( )</h1>
```

Comment:
i think this should be sorted before species, after Promise.resolve

Reply:
Yeah, the spec is fairly consistent that, when ordering properties by key, Strings collate before Symbols. (`Properties of the RegExp Prototype Object` has the only exceptions, I think.)